lighthouse-core/lib/i18n/i18n.js

```diff
@@ -184,21 +184,31 @@ const _ICUMsgNotFoundMsg = 'ICU message not found in destination locale';
 function _formatIcuMessage(locale, icuMessageId, fallbackMessage, values) {
   const localeMessages = LOCALES[locale];
   if (!localeMessages) throw new Error(`Unsupported locale '${locale}'`);
+  let localeMessage = localeMessages[icuMessageId] && localeMessages[icuMessageId].message;
 
-  const localeMessage = localeMessages[icuMessageId] && localeMessages[icuMessageId].message;
   // fallback to the original english message if we couldn't find a message in the specified locale
   // better to have an english message than no message at all, in some number cases it won't even matter
-  const messageForMessageFormat = localeMessage || fallbackMessage;
-  if (messageForMessageFormat === undefined) throw new Error(_ICUMsgNotFoundMsg);
+  if (!localeMessage) {
```

Comment:
talked about this in person, and I think we generally agree that:
- we should continue to use `fallbackMessage` over an explicit `en` lookup, since that allows development (changing `UIStrings` values) without a build step (running `yarn i18n:collect-strings`)
- but we should rename `fallbackMessage` since it's not a mysteriously-determined fallback, it's explicitly the  original UIStrings string since that's what's used as the `str_` key
- and we should add a warning to tell the dev to run `yarn i18n:collect-strings` if the (renamed) `fallbackMessage` doesn't match the en one

Reply:
Better?